### PR TITLE
ST update

### PR DESCRIPTION
**Pull request by MarcusJoh**

### CODE REVIEW

File: examples/ConnectWithWEP/ConnectWithWEP.ino [hunk 1]
```diff
@@ -25,8 +25,10 @@
 #include <SPI.h>
 #include <WiFi101.h>
 
-char ssid[] = "yourNetwork";                     // your network SSID (name)
-char key[] = "D0D0DEADF00DABBADEAFBEADED";       // your network key
+#include "arduino_secrets.h" 
+///////please enter your sensitive data in the Secret tab/arduino_secrets.h
+char ssid[] = SECRET_SSID;        // your network SSID (name)
+char pass[] = SECRET_PASS;    // your network password (use for WPA, or use as key for WEP)
```
Comment (per1234) [inline examples/ConnectWithWEP/ConnectWithWEP.ino]: This should be:
```arduino
char key[] = SECRET_PASS;
```

**Comment by per1234** [comment]
I would recommend you to use better commit descriptions. A few seconds of your time can save much headache for the innumerable people who will be browsing the commit history of these repositories in the years to come wondering what "ST update" means.

**Comment by ogiewon** [comment]
I was wondering the exact same thing!  What does ST Update mean?

**Comment by mbanzi** [comment]
It's for an upcoming new feature that will be

**Pull request merged**